test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -448,12 +459,17 @@ def test_multisched_support(self):
             self.skip_test("pbs_snapshot not found")
 
         # Setup 2 schedulers
-        sched_ids = ["sc1", "sc2", "default"]
+        sched_ids = ["sc1", "sc2", "sc3", "default"]
         self.setup_sc(sched_ids[0], "P1", "15050")
         self.setup_sc(sched_ids[1], "P2", "15051")
+        # Setup scheduler at non-default location
+        dir_path = os.path.join(os.sep, 'var', 'spool', 'pbs', 'sched_dir')
```

Comment:
the default location is also PBS_HOME, so I think it'd be a better test if the non-default location was somewhere outside of PBS_HOME, I'd recommend putting it under /tmp and see if it pbs_snapshot can still capture it correctly.

Reply:
Also, just set the complete path here (including 'sched_priv_sc3' and 'sched_logs_sc3') instead of hard-coding it inside setup_sc().

Reply:
Can you just create it inside the user's home directory (not PBS_HOME, just the user's home/?

Reply:
ok, i will try to create inside user's home and let you know.Thanks

Reply:
I had tried to create a directory inside user's home but couldn't start the scheduler. I had discussed with @suresh-thelkar so as per the current design, we can create it inside only such user's home whose have "uid" < "10" . @suresh-thelkar Please correct if i got it wrong.

Reply:
ok, I'm fine with keeping the code as it is (inside PBS_HOME/<dir>)